src/Simulator/Simulator/XamlInspection/XamlSinglePropertyEditor.xaml.cs

```diff
@@ -62,19 +86,27 @@ void Refresh()
         {
             try
             {
-                ButtonOK.Visibility = Visibility.Collapsed;
+                Type propertyType = _propertyInfo.PropertyType;
```

Comment:
Set the property name for read only properties too.
```suggestion
                Type propertyType = _propertyInfo.PropertyType;
                this.PropertyNameTextBlock.Text = _propertyInfo.Name + ":";
```